### DISCUSSION TITLE
iOS cross-platform PasswordChar is set to '●', is not working

### DISCUSSION THREAD

**Discussion by Rushikesh3009** [AvaloniaUI/Avalonia#13633] [Q&A]
PasswordChar ='●' is working good in Window and android but when I add Password Char property to iOS, application get crash.

My Following code is:
`<TextBox Name="Password" PasswordChar='●' />`

**Comment by timunie** [comment]
do other char work? Is it the symbol?

**Reply by Rushikesh3009** [reply]
I also used ' * ' , But it still got crash in iPad.

I think there should be simple solution for it because in window and android its working properly

**Reply by thevortexcloud** [reply]
Do you have a stack trace somebody could look at?

**Reply by timunie** [reply]
I have no I phone but could be an actual bug. Maybe you want to open an issue for it with a minimal sample attached
